README.md

```diff
@@ -29,97 +29,6 @@ One of the very common use cases of WebXR is 180 and/or 360 photos or videos, bo
 Another example where composition layers are going to shine is displaying text or high resolution textures in the virtual world. Since composition layers allow to sample source texture at full resolution without downsampling to the resolution of the eye buffers (which is usually much lower than the physical resolution of the device's screen(s)) the readability of the text is going to be significantly improved.
```

Comment:
>  usually much lower than the physical resolution of the device's screen(s)

FWIW, this phrasing seems a bit mobile-centric. On desktop VR systems, the eye buffers are in some cases higher resolution than the device screen. The overall point is of course still valid, and there's a quality hit from repeated resampling.

Reply:
Agreed. Will make the change to wording.